src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+// books.js
+
+// Actions... types
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] I think here it would be great if you create the action in the duck's patterns as existed in the requirements Write your book's actions and reducer (in /src/redux/books/books.js) using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).

- You can write something like this 
> const ADD_BOOK = bookstore/books/ADD_BOOK
> const REMOVE_BOOK = bookstore/books/REMOVE_BOOK